js/apps/filter-page/directives/filters.js

```diff
@@ -10,32 +10,35 @@
 
 				if ($scope.filterFamilies && $scope.filterFamilies.length) {
 					$scope.filterFamilies.forEach((filterFamily) => {
-						const filterFamilyHasTags = filterFamily && Object.hasOwnProperty.call(filterFamily, 'tags') && filterFamily.tags.length;
-						const tags = filterFamilyHasTags ? 
-							filterFamily.tags : 
-							[];
-						let hasMatch = false;
-						
-						$scope.activeFilters.forEach((filter) => {
-							if (!hasMatch) {
-								hasMatch = !!tags
-									.filter((tagName) => findFilterMatch(tagName, filter)).length;
-							}
-						});
-
-						filterFamily.isFilterActive = hasMatch;
+						if (filterFamily) {
```

Comment:
Could be rewritten in order to maintain the same indentation level...

```
if (!filterFamily) {
    return;
}

// ...
```

Reply:
@ascire I like this better, changing.